### ISSUE TITLE
I used this Library for picking Photo. But problem is After picking in Android it is showing preview Screen, Whether in iOS it is not showing any preview Screen. Can you please check it and let me know. What is the issue here?

### ISSUE THREAD

**Issue by sauravchaudhary94** [jamesmontemagno/MediaPlugin#701]
Make sure you follow all of the setup directions before posting an issue: https://github.com/jamesmontemagno/MediaPlugin#important-permission-information

If you are creating an issue for a BUG please fill out this information. If you are asking a question or requesting a feature you can delete the sections below. 

**Failure to fill out this information will result in this issue being closed.** If you post a full stack trace in a bug it will be closed, please post it to http://gist.github.com and then post the link here.

## Bug Information

Version Number of Plugin:
Device Tested On:
Simulator Tested On:
Version of VS: 
Version of Xamarin:
Versions of other things you are using:

**Comment by mrakhra** [comment]
I am looking into this issue as well. This will be good, if preview screen (white blank screen) can be removed after selecting images in Android.

**Comment by sauravchaudhary94** [comment]
Yes It will be good If it will get removed . Then manually we can add preview screen If we want.
